roles/oneview_label_facts/README.md

```diff
@@ -0,0 +1,41 @@
+## oneview_interconnect_facts
+Retrieve facts about one or more of the OneView Interconnects.
```

Comment:
replace Interconnects with Labels. Also change the heading

Reply:
done